WcaOnRails/config/locales/de.yml

```diff
@@ -5430,3 +5449,22 @@ de:
     privacy: Datenschutz
     #original_hash: 4c4a2e8
     disclaimer: Disclaimer
+  api:
+    #original_hash: 4ab4ae5
+    manage_oauth_html: 'Gehe zu %{oauth_link}, um deine OAuth Anwendungen zu verwalten'
+    #original_hash: d965ad1
+    token_request_intro: >-
+      Hier ist ein Beispiel, welches curl verwendet, um ein Access-Token
+      mithilfe von Application ID, Client Secret, deiner WCA E-Mail, und deinem
```

Comment:
I've noticed the German translation for Client secret is a bit clunky for the oath -> applications -> secret. It just says "Geheim", can you change that to also say Client Secret to have parity between these to texts?

Reply:
Ok, I can do that, "Geheim" sound a bit weird indeed.
Is is possible to amend the current PR? (As it has been created by thewca-bot and not myself)?

Reply:
Not sure how well versed you are in git, but you can add commits by  pushing to the [translation-de-556f3cce472ad94442dd82520d5bead8f7c9cb1a](https://github.com/thewca-bot/worldcubeassociation.org/tree/translation-de-556f3cce472ad94442dd82520d5bead8f7c9cb1a) branch on [thewca-bot/worldcubeassociation.org](https://github.com/thewca-bot/worldcubeassociation.org).

Reply:
or maybe only I can do that :P

Reply:
just make another one after this one is finished if you can't

Reply:
I will try, if that works as "Contributor" (although I thought one gets only the remote to pull, not to push to in that specific case).

Reply:
So it says
`ERROR: Permission to thewca-bot/worldcubeassociation.org.git denied to AnnikaStein.`
after checking out the correct branch, making the change, committing locally and then trying to push to the remote branch. Then we'll go for the second option and wait after this PR is integrated. :)